sdk/batch/Azure.Compute.Batch/samples/Sample1_CreatePool_Job_Task.md

```diff
@@ -0,0 +1,241 @@
+# Creating a Batch Pool, Job, and Tasks
+
+This sample demonstrates how to create a pool, job, and tasks for a Batch account.
+
+## Special Note for Batch Pools
+
+Both the **Azure.Compute.Batch** and the ARM based **Azure.Management.Batch** sdk's support the creating and managing of Batch Pools with the only difference being that **Azure.Management.Batch** supports creating [Batch Pools with Managed Idenities](https://learn.microsoft.com/azure/batch/managed-identity-pools), for this reason creating pools via **Azure.Management.Batch** is considered a best practice.  This example will use **Azure.Management.Batch** for pool creation and thus will need to create an ArmClient to do so.
+
+## Authenticating the Azure.ResourceManager `ArmClient`
+
+In order to create Batch Pool from the Azure.Managment.Batch library you will need to instantiate an Armclient. To create an authenticated client and start interacting with Microsoft Azure resources, see the [quickstart guide here](https://github.com/Azure/azure-sdk-for-net/blob/main/doc/dev/mgmt_quickstart.md).
+
+```C#
```

Comment:
As mentioned in the README, these need to be snippets.

Reply:
done